### DISCUSSION TITLE
如何解决 Taro 暂未实现 event 的 capture 特性这个警告

### DISCUSSION THREAD

**Discussion by daiwanxing** [NervJS/taro#16030] [Q & A] [answered]
![image](https://github.com/NervJS/taro/assets/37327614/a2be882d-4c31-404f-8458-f0b2b8f975ca)

如上图所示：这是我在 Taro 内使用第三方组件库的时候控制台输出的警告日志，请问如何修复这个警告

**Comment by daiwanxing** [comment, marked as answer]
没人回答，我自己回答好了。。看了源码这个异常的抛出是因为 Taro 自己写的一套 `EventListener` 并不支持在事件捕获阶段执行。

**Reply by Damiyinliao** [reply]
请问你是如何消除这个警告的？

**Reply by daiwanxing** [reply]
> 请问你是如何消除这个警告的？

没消除的方法，也没关系，就是控制台输出难看。
